File: examplefile.py
def fibonacci(n):
    if n <= 0:
        return "Input must be a positive integer"
    elif n == 1:
        return 0
    elif n == 2:
        return 1
    
    a, b = 0, 1
    for _ in range(n - 2):
        a, b = b, a + b
    return b

File: examplefile.cpp
void fibonacci(int n) {
    if (n <= 0) {
        return "Input must be a positive integer";
    }
    
    int a = 0, b = 1;
    for (int i = 2; i <= n; ++i) {
        int c = a + b;
        a = b;
        b = c;
    }
    
    return b;
}